test/e2e/grpc/server.test.ts

```diff
@@ -130,6 +132,19 @@ describe("grpc/server", () => {
       expect(mockPubSub.mock.calls[0][0]["hello"]).toEqual("world");
     });
 
+    it("should be able to receive events with their respective headers", async () => {
+      await server.client.pubsub.publish("pubsub-redis", "topic-5", { hello: "world" });
+
+      // Delay a bit for event to arrive
+      await new Promise((resolve, _reject) => setTimeout(resolve, 250));
+
+      expect(mockPubSubWithHeaders.mock.calls.length).toBe(1);
+
+      // Also test for receiving data
```

Comment:
Same here, to check for data.

Reply:
I don't think so seeing that there are 3 checks under it so the test will fail in any case